runtime/bus.go

```diff
@@ -71,6 +71,10 @@ func (m *bus) GetPersistenceModule() modules.PersistenceModule {
 	return getModuleFromRegistry[modules.PersistenceModule](m, modules.PersistenceModuleName)
 }
 
+func (m *bus) GetTreeStoreModule() modules.TreeStoreModule {
```

Comment:
Is this the reason you are making TreeStore a full module? If so #855 addresses this and allows Submodules to be called in this way